README.md.pre-ubuntu18

```diff
@@ -0,0 +1,338 @@
+# ENCODE Metadata Database
+-------------------------------------------------------
+
+
+[![CircleCI](https://circleci.com/gh/ENCODE-DCC/encoded/tree/dev.svg?style=svg)](https://circleci.com/gh/ENCODE-DCC/encoded/tree/dev)
+
+## Setting Up Your Environment
+
+These are the primary software versions used in production, and you should be able to use them locally:
```

Comment:
I do not see the point of mentioning this in that these Encode will not work without even one of the other dependencies.

Reply:
These were the original instructions.  I didn't edit it, just moved the file.